src/Internal/ProductAttributesLookup/Filterer.php

```diff
@@ -196,7 +196,9 @@ private function get_product_counts_query_using_lookup_table( $tax_query, $meta_
 
 		$query['select'] = 'SELECT COUNT(DISTINCT product_or_parent_id) as term_count, term_id as term_count_id';
 		$query['from']   = "FROM {$this->lookup_table_name}";
-		$query['join']   = "INNER JOIN {$wpdb->posts} ON {$wpdb->posts}.ID = {$this->lookup_table_name}.product_or_parent_id";
+		$query['join']   = "
```

Comment:
With this format I believe the `INNER JOIN` won't get applied if `{$tax_query_sql['join']} {$meta_query_sql['join']}` is empty.

Reply:
You mean the line format? I put it that way for readability. If these are empty then only the `INNER JOIN {$wpdb->posts}` right below will apply.